tests/bin/eslint.js

```diff
@@ -221,11 +221,8 @@ describe("bin/eslint.js", () => {
                 return assertExitCode(child, 0).then(() => {
                     assert.isTrue(fs.existsSync(CACHE_PATH), "Cache file should exist at the given location");
 
-                    assert.doesNotThrow(
-                        () => JSON.parse(fs.readFileSync(CACHE_PATH, "utf8")),
-                        SyntaxError,
-                        "Cache file should contain valid JSON"
-                    );
+                    // "Cache file should contain valid JSON"
+                    JSON.parse(fs.readFileSync(CACHE_PATH, "utf8"));
```

Comment:
I think it would be better to move the `assert.doesNotThrow` changes to a separate PR. In that PR, it would probably be a good idea to add it to the [linting configuration](https://github.com/eslint/eslint/blob/218ee57deb5eaec6c9df6fbf45d10673b51a2c2b/packages/eslint-config-eslint/default.yml#L93-L101) too, so that no one adds additional `assert.doesNotThrow` calls in the future.